amplify/backend/function/validateUserInvite/src/index.js

```diff
@@ -117,7 +97,30 @@ exports.handler = async (event) => {
   );
 
   if (contributorExistenceCheckQuery?.data?.getColonyContributor) {
-    // contributor already exists no need to create one
+    // If the user has already been a contributor for a Colony before,
+    // switching their isWatching field to true will allow them to access it again
+    const colonyMemberIsWatchingMutation = await graphqlRequest(
+      updateColonyContributor,
+      {
+        input: {
+          id: getColonyContributorId(colonyAddress, userAddress),
+          isWatching: true,
+        },
+      },
+      graphqlURL,
+      apiKey,
+    );
+
+    if (
+      colonyMemberIsWatchingMutation.errors ||
+      !colonyMemberIsWatchingMutation.data
+    ) {
+      throw new Error(
+        colonyMemberIsWatchingMutation?.message ||
```

Comment:
We ought to get TypeScript into lambdas at some point to catch those early. 

I think this should be:
```ts
colonyMemberIsWatchingMutation.errors[0]?.message
```

Reply:
Ah yes.... my bad 🤦‍♂️